### PR TITLE
Python-images-updates

### PR DESCRIPTION
**Pull request by cwebster-99**
Updating Python related images to reflect up to date UI

### CODE REVIEW

File: docs/python/python-tutorial.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ Area: python
 TOCTitle: Tutorial
 ContentId: 77828f36-ae45-4887-b25c-34545edd52d3
 PageTitle: Get Started Tutorial for Python in Visual Studio Code
+
```
Comment (cwebster-99) [inline docs/python/python-tutorial.md:7]:
```suggestion
```

File: docs/python/python-tutorial.md [hunk 2]
```diff
@@ -4,7 +4,9 @@ Area: python
...
 DateApproved: 1/20/2023
+
```
Comment (cwebster-99) [inline docs/python/python-tutorial.md:9]:
```suggestion
```

File: docs/python/python-tutorial.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ This tutorial introduces you to VS Code as a Python environment - primarily how
 
 This tutorial is not intended to teach you Python itself. Once you are familiar with the basics of VS Code, you can then follow any of the [programming tutorials on python.org](https://wiki.python.org/moin/BeginnersGuide/Programmers) within the context of VS Code for an introduction to the language.
 
-If you have any problems, you can search for answers or ask a question on the [Python extension Discussions Q&A](https://github.com/microsoft/vscode-python/discussions/categories/q-a).
+If you have any problems, feel free to file an issue for this tutorial in the [VS Code documentation repository](https://github.com/microsoft/vscode-docs/issues).
```
Comment (cwebster-99) [inline docs/python/python-tutorial.md:23]:
```suggestion
If you have any problems, you can search for answers or ask a question on the [Python extension Discussions Q&A](https://github.com/microsoft/vscode-python/discussions/categories/q-a).
```

File: docs/python/python-tutorial.md [hunk 4]
```diff
@@ -45,8 +45,10 @@ Along with the Python extension, you need to install a Python interpreter. Which
 
 Install [Python from python.org](https://www.python.org/downloads/). You can typically use the **Download Python** button that appears first on the page to download the latest version.
 
+
```
Comment (cwebster-99) [inline docs/python/python-tutorial.md:48]:
```suggestion
```

File: docs/python/python-tutorial.md [hunk 5]
```diff
@@ -45,8 +45,10 @@ Along with the Python extension, you need to install a Python interpreter. Which
...
 >**Note**: If you don't have admin access, an additional option for installing Python on Windows is to use the Microsoft Store. The Microsoft Store provides installs of [supported Python versions](https://apps.microsoft.com/store/search?publisher=Python%20Software%20Foundation).
 
+
```
Comment (cwebster-99) [inline docs/python/python-tutorial.md:51]:
```suggestion
```

File: docs/python/python-tutorial.md [hunk 6]
```diff
@@ -63,7 +65,7 @@ The built-in Python 3 installation on Linux works well, but to install other Pyt
...
+- **Windows Subsystem for Linux**: If you are working on Windows and want a Linux environment for working with Python, the [Windows Subsystem for Linux](https://docs.microsoft.com/windows/wsl/about) (WSL) is an option for you. If you choose this option, you'll also want to install the [Remote - WSL extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl). For more information about using WSL with VS Code, see [VS Code Remote Development](/docs/remote/remote-overview.md) or try the [Working in WSL tutorial](/docs/remote/wsl-tutorial.md), which will walk you through setting up WSL, installing Python, and creating a Hello World application running in WSL.
```
Comment (cwebster-99) [inline docs/python/python-tutorial.md:68]:
```suggestion
- **Windows Subsystem for Linux**: If you are working on Windows and want a Linux environment for working with Python, the [Windows Subsystem for Linux](https://learn.microsoft.com/windows/wsl/about) (WSL) is an option for you. If you choose this option, you'll also want to install the [WSL extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl). For more information about using WSL with VS Code, see [VS Code Remote Development](/docs/remote/remote-overview.md) or try the [Working in WSL tutorial](/docs/remote/wsl-tutorial.md), which will walk you through setting up WSL, installing Python, and creating a Hello World application running in WSL.
```

File: docs/python/python-tutorial.md [hunk 7]
```diff
@@ -77,7 +79,7 @@ To verify that you've installed Python successfully on your machine, run one of
 
 - Windows: open a command prompt and run the following command:
 
-    ```bat
+    ```ps
```
Comment (cwebster-99) [inline docs/python/python-tutorial.md:82]:
```suggestion
    ```bat
```

File: docs/python/python-tutorial.md [hunk 8]
```diff
@@ -228,7 +229,7 @@ For full details, see [Debugging configurations](/docs/python/debugging.md), whi
 
 ## Install and use packages
 
-Let's now run an example that's a little more interesting. In Python, packages are how you obtain any number of useful code libraries, typically from [PyPI](https://pypi.org/). For this example, you use the `matplotlib` and `numpy` packages to create a graphical plot as is commonly done with data science. (Note that `matplotlib` cannot show graphs when running in the [Windows Subsystem for Linux](https://learn.microsoft.com/windows/wsl/about) as it lacks the necessary UI support.)
+Let's now run an example that's a little more interesting. In Python, packages are how you obtain any number of useful code libraries, typically from [PyPI](https://pypi.org/). For this example, you use the `matplotlib` and `numpy` packages to create a graphical plot as is commonly done with data science. (Note that `matplotlib` cannot show graphs when running in the [Windows Subsystem for Linux](https://docs.microsoft.com/windows/wsl/about) as it lacks the necessary UI support.)
```
Comment (cwebster-99) [inline docs/python/python-tutorial.md:232]:
```suggestion
Let's now run an example that's a little more interesting. In Python, packages are how you obtain any number of useful code libraries, typically from [PyPI](https://pypi.org/). For this example, you use the `matplotlib` and `numpy` packages to create a graphical plot as is commonly done with data science. (Note that `matplotlib` cannot show graphs when running in the [Windows Subsystem for Linux](https://learn.microsoft.com/windows/wsl/about) as it lacks the necessary UI support.)
```

File: docs/python/python-tutorial.md [hunk 9]
```diff
@@ -278,6 +279,7 @@ v
 1. Ensure your new environment is selected by using the **Python: Select Interpreter** command from the **Command Palette**.
 
    ![Select an Interpreter](images/tutorial/interpreter-venv.png)
+
```
Comment (cwebster-99) [inline docs/python/python-tutorial.md:282]:
```suggestion
```

File: docs/python/python-tutorial.md [hunk 10]
```diff
@@ -320,5 +322,6 @@ There is then much more to explore with Python in Visual Studio Code:
 - [Debugging](/docs/python/debugging.md) - Learn to debug Python both locally and remotely.
 - [Testing](/docs/python/testing.md) - Configure test environments and discover, run, and debug tests.
 - [Settings reference](/docs/python/settings-reference.md) - Explore the full range of Python-related settings in VS Code.
+
```
Comment (cwebster-99) [inline docs/python/python-tutorial.md:325]:
```suggestion
```

File: docs/python/python-tutorial.md [hunk 11]
```diff
@@ -99,7 +99,6 @@ code .
 
 >**Note**: If you're using an Anaconda distribution, be sure to use an Anaconda command prompt.
```
Comment (cwebster-99) [inline docs/python/python-tutorial.md:101]:
```suggestion


```

**Comment by luabud** [comment]
This looks great to me, thank you!

The one thing I'm wondering is whether we should hide the icons in the activity bar that don't come by default (e.g. GitHub, live share, etc.), just so users don't get confused in case they think the icons should show up as they follow along the documentation. @gregvanl do you have thoughts/guidance on this?

**Comment by gregvanl** [comment]
Yes, it is better to have a clean Stable build without extra customizations for taking screenshots.
With the new Profiles feature, you can create an Empty Profile (File > Preferences > Profiles > Create Profile then select Create an Empty Profile) to have a clean editor.

I'm confused by a couple of the images not rendering in the PR. Any idea what's up with new-file-icon.png and debug-panel-initial-view.png? You might try updating them again. All the other images render correctly on GitHub and are correctly stored in Git LFS

**Comment by cwebster-99** [comment]
Good feedback! I just updated the images to clean up the activity bar and then reuploaded the images that did not render.

**Review by luabud — APPROVED** [review]

**Pull request merged**